src/resources/wifi_esp32.cc

```diff
@@ -604,6 +589,35 @@ PRIMITIVE(read_scan) {
 
   return ap_array;
 }
+
+PRIMITIVE(ap_info) {
+  ARGS(WifiResourceGroup, group);
+
+  wifi_ap_record_t ap_record;
+  esp_err_t ret = esp_wifi_sta_get_ap_info(&ap_record);
+  if (ret != OK) return Primitive::os_error(ret, process);
+
+  const size_t element_count = 5;
+  Array* ap_array = process->object_heap()->allocate_array(element_count, Smi::zero());
+  if (ap_array == null) ALLOCATION_FAILED;
+
+  String* ssid = process->allocate_string((char *)ap_record.ssid);
+  if (ssid == null) ALLOCATION_FAILED;
+
+  size_t bssid_size = 6;
```

Comment:
For consistency with `element_count`:

```suggestion
  const size_t bssid_size = 6;
```